packages/client/hmi-client/src/services/model-configurations.ts

```diff
@@ -249,6 +265,63 @@ export function setParameterSource(
 	}
 }
 
+export function getParameterConstant(config: ModelConfiguration, parameterId: string): number {
+	const parameter = getParameter(config, parameterId);
+	return parameter?.value ?? NaN;
+}
+
+export function setParameterConstant(
+	config: ModelConfiguration,
+	parameterId: string,
+	value: string
+): void {
+	const parameter = getParameter(config, parameterId);
+	if (parameter) {
+		parameter.value = parseFloat(value);
+	}
+}
+
+// sets the parameter distribution type along with its default respective parameters
+export function setParameterDistributionType(
+	config: ModelConfiguration,
+	parameterId: string,
+	distributionType: DistributionType
+): void {
+	const parameter = getParameter(config, parameterId);
+	if (!parameter) return;
+	switch (distributionType) {
+		case DistributionType.Constant:
+			delete parameter.distribution;
+			break;
+		case DistributionType.Uniform:
+			parameter.distribution = {
+				type: DistributionType.Uniform,
+				parameters: {
+					minimum: 0,
+					maximum: 0
+				}
+			};
+			break;
+		default:
```

Comment:
Im doing some additional tests here on how this value/distribution should be populated when changing the dropdown:
https://github.com/DARPA-ASKEM/terarium/issues/3700